geoslurp/db/connector.py

```diff
@@ -168,4 +168,13 @@ def addUser(self,name,passw,readonly=False):
 
 
 
+    #ay, adapted from geoslurptools.db.connector
+    def getInvent(self,tname,scheme):
```

Comment:
This is a useful function. I would suggest changing the name to `getInventEntry` as it returns only a row of the inventory and not the entire inventory itself.
```suggestion
    def getInventEntry(self,tname,scheme):
```